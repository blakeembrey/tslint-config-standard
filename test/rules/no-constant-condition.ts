if (true) {
  console.log('test')
}

while (true) {
  if (check()) {
    break
  }
}

export function check () {
  return true
}
