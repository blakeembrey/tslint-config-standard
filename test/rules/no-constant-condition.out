
no-constant-condition.ts[1, 5]: unexpected constant condition
